src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonNull.cs

```diff
@@ -42,24 +42,44 @@ public sealed class JsonNull : JsonNode, IEquatable<JsonNull>
         ///   Compares other JSON null to the value of this instance.
         /// </summary>
         /// <param name="other">The JSON null to compare against.</param>
-        /// <returns><see langword="true"/></returns>
-        public bool Equals(JsonNull other) => true;
+        /// <returns>
+        ///    <see langword="true"/> if the <paramref name="other"/> is <see cref="JsonNull"/>,
```

Comment:
This comment doesn't match the API. `other` is already a `JsonNull`.

```suggestion
        ///    <see langword="true"/> if <paramref name="other"/> is not null,
```